### PR TITLE
add back in stock support

### PR DESCRIPTION
**Pull request by ronnyrin**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -27,6 +31,14 @@
...
     ]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "/Users/ronnyr/Projects/wix-commerce-ticketing-nextjs-template/.next/types/**/*.ts"
```
Comment (barak007) [inline tsconfig.json:39]: @ronnyrin Hey this is probably not intentional.

File: app/constants.ts [hunk 2]
```diff
@@ -4,3 +4,5 @@ export const WIX_ACCESS_TOKEN = 'wix_accessToken';
 export const WIX_REFRESH_TOKEN = 'wix_refreshToken';
 
 export const PLACEHOLDER_IMAGE = '/images/placeholder.jpg';
+
+export const STORES_APP_ID = '1380b703-ce81-ff05-f115-39571d94dfcd';
```
Comment (barak007) [inline app/constants.ts:8]: not sure about this...also

File: tsconfig.json (hunk 1 above)
Comment (ronnyrin) [reply tsconfig.json:39]: lol

File: app/constants.ts (hunk 2 above)
Comment (ronnyrin) [reply app/constants.ts:8]: it's from https://dev.wix.com/docs/sdk/api-reference/ecom/cart/add-to-cart

**Pull request merged**